docs/design/Colors.stories.mdx

```diff
@@ -84,6 +84,18 @@ The default color used for interactive elements.
 Use for interactive elements that should be less visually pronounced, such as an
 icon action in main navigation, or buttons to dismiss or cancel a workflow.
 
+##### Interactive--Background
+
+<Swatch
+  colors={[
+    "--color-interactive--background",
+    "--color-interactive--background--hover",
+  ]}
+/>
+
+Use for the background of interactive elements to ensure separation from the
+surface they appear on.
+
```

Comment:
When we get back to that "automatically pull in the description from the JSON" task we get to skip this step in the future 😄